machine-learning/app/models/base.py

```diff
@@ -186,7 +186,7 @@ def provider_options_default(self) -> list[dict[str, Any]]:
                 case "CPUExecutionProvider" | "CUDAExecutionProvider":
                     option = {"arena_extend_strategy": "kSameAsRequested"}
                 case "OpenVINOExecutionProvider":
-                    option = {"device_type": "GPU_FP32"}
+                    option = {"device_type": "GPU_FP32", "cache_dir": f"{self.cache_dir}/openvino"}
```

Comment:
Since `self.cache_dir` is a `Path`, you should do forward slashes instead of string concatenation:
```suggestion
                    option = {"device_type": "GPU_FP32", "cache_dir": (self.cache_dir / "openvino").as_posix()}
```